webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMS.java

```diff
@@ -10,6 +10,14 @@
  */
 interface ThirdPartyTMS {
 
+  /**
+   * Removes a context from the third party TMS
+   *
+   * @param projectId the third party project id
+   * @param contextId the context id
+   */
+  void removeContext(String projectId, String contextId);
```

Comment:
That should probably a "removeImage" if current API just allows to "uploadImage" (behind the scene Smartling has the notion of context but that is not visible in this API abstraction)

Reply:
I'll keep the name under the smartlingClient as removeContext then, but I'll change all the other references until that call to removeImage